### PR TITLE
Space brigade logic

### PR DESCRIPTION
**Pull request by YehorFeshchenko**
Moving space brigade

### CODE REVIEW

File: src/game/SpaceBrigade.ts [hunk 1]
```diff
@@ -10,9 +10,35 @@ export interface SpaceBrigadeProps extends SpaceEntity {
...
     Object.assign(this, props)
     makeAutoObservable(this, undefined, { autoBind: true })
+    this.startMovement(Date.now())
+  }
+
+  private startMovement(startTime: number) {
+    const animate = () => {
+      const elapsedTime = (Date.now() - startTime) / 1000; // Convert milliseconds to seconds
+      this.updatePosition(elapsedTime);
+      if (elapsedTime < 10) { // Continue for the duration of 10 seconds
```
Comment (floumaster) [inline src/game/SpaceBrigade.ts]: we dont need to interrupt the journey on fe. Once the be sends us, that space brigade is reached the planet, we will delete the class instance and the loop will stop.

File: src/game/SpaceBrigade.ts [hunk 2]
```diff
@@ -10,9 +10,35 @@ export interface SpaceBrigadeProps extends SpaceEntity {
...
+          this.animationFrameId = null;
+        }
+      }
+    };
+    this.animationFrameId = requestAnimationFrame(animate);
+  }
+
+  private updatePosition(elapsedTime: number) {
+    const duration = 10; // Duration to complete the journey in seconds
+    const fraction = Math.min(elapsedTime / duration, 1);
```
Comment (floumaster) [inline src/game/SpaceBrigade.ts]: This duration should not be a const value. It should be calculated as distance(planet1.coordinate, planet2.coordinate) / spaceBrigadeSpeed

File: src/game/Game.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ import { GameApi } from "./api/GameApi";
 import { LobbyFactory, LobbyApi, Lobby } from "lobby";
 
 class Game {
-  private _state: GameState = 'initial'
+  private _state: GameState = 'inProgress'
```
Comment (floumaster) [inline src/game/Game.ts:12]: discard this change

File: .gitignore [hunk 4]
```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+#ide
```
Comment (floumaster) [inline .gitignore:25]: discard this change

**Comment by YehorFeshchenko** [comment]
@floumaster Changes we discussed privately are pushed, tell me if there are more modifications needed

**Review by floumaster — APPROVED** [review]

**Pull request merged**